mkdocs.yml

```diff
@@ -47,5 +47,5 @@ nav:
         - "_naming_strategies-underscore_mapping": naming-strategy/underscore-naming-strategy.md
         - "_naming_strategies-composite": naming-strategy/composite-naming-strategy.md
 site_name: laminas-hydrator
-site_description: "Serialize objects to arrays, and vice versa"
+site_description: "laminas-hydrator provides functionality for hydrating objects and extracting data from them. The component contains concrete implementations for a number of common use cases and provides interfaces for creating custom implementations."
```

Comment:
All descriptions should have the same content:

* Component documentation: https://docs.laminas.dev/laminas-hydrator/ (`mkdocs.yml`)
* Component overview: https://docs.laminas.dev/components/
* Packagist: https://packagist.org/packages/laminas/laminas-hydrator ([`composer.json`](https://github.com/laminas/laminas-hydrator/blob/3.2.x/composer.json#L3))
* GitHub repository: https://github.com/laminas/laminas-hydrator

Reply:
You sure are thorough. I'm feeling fortunate to have you review my work.

Reply:
If you're happy with me making the change to the project's about info, I'll get it done, otherwise I'll wait until the PR's merged.

Reply:
Unfortunately the description is too long:

![Bildschirmfoto 2020-09-23 um 07 49 45](https://user-images.githubusercontent.com/103362/93971957-a0da1500-fd71-11ea-9fd8-028186c2b203.png)
![Bildschirmfoto 2020-09-23 um 07 50 34](https://user-images.githubusercontent.com/103362/93971960-a33c6f00-fd71-11ea-8d45-49a500e56ca6.png)

But I have another suggestion: we should add an introduction page where the description can be added. This would also for the "Quick Start" page because the quick start should explain the usage and not the component itself.